auth.js

```diff
@@ -1,28 +1,40 @@
 import { AuthorizationCode } from "simple-oauth2";
 
-const adminConfiguration = {
-  // These credentials come from your .env file.
-  clientID: process.env.ADMINISTRATION_CLIENT_ID,
-  clientSecret: process.env.ADMINISTRATION_CLIENT_SECRET
-};
+// This function can be used by callers to retrieve a token prior to their API calls.
+/**
+ * @param {Object} config - A configuration object for authorizing the API client.
+ * @param {string} config.clientId - The client ID for the API client.
+ * @param {string} config.clientSecret - The client secret for the API client.
+ * @param {string} config.audience - The audience associated with the target API.
+ * @returns {string}
+ */
+export async function getAccessToken(config) {
```

Comment:
I moved this function to the top of the file, because it is the entry point to this logic, and for users who want to read the code and understand how to use a library to handle auth for their code, this is where they'll want to start.

It also now takes a `config` object, which contains three properties: `clientId`, `clientSecret`, and `audience`. Previously we were passing a magic string and doing some lookups within this file to get the correct environment variables, but that was obfuscating important details -- you need the client id, secret, and audience in order to authorize.